src/initiatives/process.md

```diff
@@ -0,0 +1,57 @@
+# Process
+
+This page describes how lang team initiatives work. This is the process to use if you have an idea for a change you would like to make in the language.
+
+## Summary
+
+In a nutshell, the process for a successful initiative is as follows:
+
+- Have an idea
+  - Talk about it on internals, Zulip, etc to flesh it out a bit
+  - Ideally, identify a potential [owner]
+- Open a [proposal] as an issue on the lang-team repository
+  - A lang team member can decide to be your [liaison] and [_second_ your proposal][2nd].
+  - Once that happens, we will create a Zulip stream, tracking issue, and (optionally) repository, etc.
+- If warranted, [explore][experimental] the design space and author the RFC
+  - In this phase, the [owner] works with the [liaison] and other contributors to expore the design space and develop the RFC
+  - Code can be landed in this phase, but the feature gate is marked as "experimental"
+  - Users of the feature gate will get a warning that the RFC is under development
+  - Once the RFC is ready, it can be opened on the RFC repository and approved by the lang team
+- Finish [development]
+  - At this point, development proceeds but the feature gate does not have to be marked as "experimental"
+- [Feature complete]
+  - When the liaison feels that it is ready, the initiative may be declared "feature complete".
+  - This is primarily a 'signaling' mechanism to encourage testing and feedback.
+  - This is a good phase in which to write the Rust reference chapter and other supporting documentation.
```

Comment:
I think this is interesting to call out -- we don't really do a great job of it today in practice, I think.

It's worth noting that for example the asm documentation lives in the unstable book right now - https://doc.rust-lang.org/nightly/unstable-book/library-features/asm.html. Maybe that's a better thing to call out here? And then, presumably, it is easy to copy out documentation from there after stabilization into the reference and other places material should live in the long run...

Reply:
We haven't historically done a great job, but I've been checking for it on recent stabilizations and we've usually had a pending reference PR. We should make a template for stabilization reports and it should include that.

I agree that we should be doing a better job developing the docs for unstable things incrementally, so that at the point where it becomes time to do the stabilization report, this is basically done. (Side note that I am not 100% convinced that having them separated out into the "unstable guide" is the right approach, versus having a "nightly" tag on the reference, but that's neither here nor there.)

Potentially this is the purpose of the "feature complete" period-- i.e., the gate for exit (or maybe entry) is 'the unstable docs are up to date".

Reply:
Entry would make sense, since it's hard to test what is not documented. At the same time, I think more important than the reference material is usually a kind of *explainer*. Maybe *that* should be the gate for entry, and reference for exit.